src/layout/components/styles/books.css

```diff
@@ -1,49 +1,137 @@
-.book_list_wrapper {
-  padding: 2rem 24px;
+.blue {
+  color: #0290ff;
+}
+
+.flex {
   display: flex;
+}
+
+.book_list_wrapper {
   flex-direction: column;
 }
 
 .book_list_wrapper ul {
-  display: flex;
   flex-direction: row;
-  padding-left: 0;
   align-items: center;
-  justify-content: space-between;
-  background-color: rgba(71, 69, 69, 0.334);
+  justify-content: flex-start;
   border-radius: 3px;
 }
 
 .book_list_wrapper ul li {
   list-style: none;
-  padding: 8px 12px;
   font-size: 14px;
 }
 
 .book_list_wrapper button {
   align-self: flex-end;
-  padding: 6px 12px;
 }
 
 @media screen and (min-width: 768px) {
-  .book_list_wrapper {
-    padding: 2rem 4rem;
-    display: flex;
-    flex-direction: column;
+  .book_list_display {
     align-items: center;
     justify-content: center;
+    padding: 0.5rem 0;
```

Comment:
*[OPTIONAL]*

- Hey, since this wasn't pointed out in the previous review, please, adjust your styles so the content can fit in this container. Currently, the content is going out of the container. see the image below:


![Screenshot from 2023-03-04 00-39-03](https://user-images.githubusercontent.com/59397159/222855694-3693d2ef-14ad-451e-a633-8c2898d7709d.png)

Reply:
@zieeco  let me fix that, Thank you!